src/components/Filters/CollapsibleLabel.tsx

```diff
@@ -1,6 +1,17 @@
 import classNames from 'classnames';
 import { ChevronIcon } from '../../icons/ChevronIcon';
 import { useFilterGroupContext } from './FilterGroupContext';
+import { useComposedCssClasses } from '../../hooks/useComposedCssClasses';
+
+/**
+ * The CSS class interface for {@link CollapsibleLabelProps}.
+ *
+ * @public
+ */
+export interface CollapsibleLabelCssClasses {
+  button?: string,
```

Comment:
I don't imagine many people would want to customize the styling for the collapse button. It might be better to leave it out of the interface and leave it hard-coded as it was before